qutip/solver/integrator/explicit_rk.pyx

```diff
@@ -122,17 +125,12 @@ cdef class Explicit_RungeKutta:
         self.interpolate = interpolate
 
         self.k = []
-        if isinstance(method, dict):
-            self._init_coeff(**method)
-        elif "vern7" == method:
-            self._init_coeff(**vern7_coeff)
-        elif "vern9" == method:
-            self._init_coeff(**vern9_coeff)
-        elif "rk4" == method:
+        if butcher_tableau is None:
             self._init_coeff(**rk4_coeff)
         else:
-            self._init_coeff(**euler_coeff)
```

Comment:
Could we keep the Euler method. I'm happy with Runge-Kutta 4th Order being the default, but it's nice to have the Euler option precisely because its so simple.

Reply:
I made the tableau input required and moved the both simple tableaux to python space.